src/Stratis.Bitcoin.Tests/Consensus/ChainedHeaderTreeTest.cs

```diff
@@ -2352,7 +2352,7 @@ public void ChainAdvancesAfterACheckpoint_NewChainIsPresentedWithForkPoint_Chain
 
                 cht.BlockDataDownloaded(currentChainTip, block);
                 cht.PartialValidationSucceeded(currentChainTip, out bool fullValidationRequired);
-                ctx.FinalizedBlockMock.Setup(m => m.GetFinalizedBlockHashAndHeight()).Returns(new HashHeightPair(uint256.One, currentChainTip.Height - maxReorg));
+                ctx.FinalizedBlockMock.Setup(m => m.GetFinalizedBlockInfo()).Returns(new HashHeightPair(uint256.One, currentChainTip.Height - maxReorg));
```

Comment:
Shouldn't you everywhere get the correct hash?

Reply:
no. tests check only heights

Reply:
i don't think you should make such assumption

Reply:
we are testing internal methods of this class. we can make such assumptions